source/documentation/getting_started/quick_start.md

```diff
@@ -1,10 +1,59 @@
 ## Quick start guide
 
-Registers in beta are reliable and are ready for use in products and services.
+Registers which are in beta are reliable and ready for use in products and services. Each register has an open, RESTful API which you can use to access the data in the register. 
 
-1. [Find the base URL for the register(s) you want to use](https://registers.cloudapps.digital/registers). For example, for the `local-authority-eng` register: `https://local-authority-eng.register.gov.uk/`.
-2. [Find the endpoints you want](#apireference). For example, `/record/BIR` in `https://local-authority-eng.register.gov.uk/record/BIR`.
-3. Add `.json`, `.yaml`, `.csv`, `.tsv`, or `.ttl` to the end of the request URL (before any query parameters) to choose the format of the response, or amend headers to `accept application/json` or equivalent.
-4. Parse register data and use it in your product or service.
+You can follow this guide to quickly get started with GOV.UK Registers APIs. 
+
+**1. Find the base URL for the register(s) you want to use.** 
+
+You can find a full list of register base URLs [here](https://registers.cloudapps.digital/registers).
+
+The following are some examples of base URLs:
+
+| Register name | Base URL |
+|----------|----------|
+| `local-authority-eng`     | `https://local-authority-eng.register.gov.uk/`|
+| `country` | `https://country.register.gov.uk/` |
+| `allergen`  | `https://allergen.register.gov.uk/` |
+
+**2. Find the endpoints you want to use.**
+
+Using different endpoints, you can:
+
+* [view information about a register](#get-register) 
+* [get all records from a register](#get-records) 
+* [find a specific record within a register](#get-record-field-value) 
+* [get all entries for a single record](#get-record-field-value-entries) 
+* [find all records that share a `field-value` for a particular field](#get-records-field-name-field-value) 
+* [get all entries from a register](#get-entries)
+* [find a specific entry from a register](#get-entry-entry-number)
+* [find a specific item within a register](#get-item-item-hash)
+* [download the full contents of a register in a ZIP file](#get-download-register) 
+
+For example, `GET record/BIR` is the endpoint for Birmingham City Council in the `local-authority-eng` register, and 'BIR' is the `field-value`. 
```

Comment:
Is this line still needed given the bullet points above?

Reply:
The idea of putting this in was to reinforce the point by example, since previous user research indicated that people struggled with the descriptions surrounding "endpoint".

Reply:
It's quite far out of context now though.  This particular example is for the third of the endpoints above, and it's the only mention of `field-value`.  I think users would be better of following the links above to find out how to use a given endpoint.

Reply:
Yes, looking at it again after all the other changes, I agree with you. cc @arnau ?